src/main/java/se/sundsvall/teamssender/auth/repository/ITokenCacheRepository.java

```diff
@@ -1,10 +1,7 @@
 package se.sundsvall.teamssender.auth.repository;
 
-import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import se.sundsvall.teamssender.auth.model.TokenCacheEntity;
 
-public interface ITokenCacheRepository {
-
-	Optional<byte[]> load(String userId);
-
-	void save(String userId, byte[] cacheData);
+public interface ITokenCacheRepository extends JpaRepository<TokenCacheEntity, String> {
```

Comment:
A circuitbreaker is recommended here.

e.g.
`@CircuitBreaker(name = "tokenCacheRepository")`

Se examples in other services that uses repositories.